### PR TITLE
Feature/migrate to google crashlytics

### PR DESCRIPTION
**Pull request by rusmonster**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,16 +8,12 @@ What you'll minimally need to get started:
...
 ## Building
 
-### Set up gradle wrapper to use correct gradle version.
+### Add google-services.json
 
-Run
-```
-./gradlew wrapper
-```
+[Generate google-services.json](https://developers.google.com/mobile/add) file and place it under `sync-quickstart-android/`.
```
Comment (berkus) [inline README.md]: Invalid url.

File: README.md [hunk 2]
```diff
@@ -39,26 +35,30 @@ You can also pass this address to gradle during the build.
...
-```
-fabricApiKey=0123456789abcdef
-```
+### Optionally setup Firebase Crashlytics
 
-You can also pass this parameter to gradle during build without need to create a properties file, as follows:
+If you want to see crashes reported to crashlytics:
+1. [Set up Crashlytics in the Firebase console](https://firebase.google.com/docs/crashlytics/get-started-new-sdk?platform=android&authuser=1#set-up-console)
 
+2. In order to see native crashes symbolicated upload symbols into the Firebase console :
```
Comment (berkus) [inline README.md]: extra space before colon

File: README.md [hunk 3]
```diff
@@ -39,26 +35,30 @@ You can also pass this address to gradle during the build.
...
 
-### Build
-
-Run `./gradlew assembleDebug` to fetch Twilio Sync SDK files and build application.
-
-### Android Studio
+Select "Simulate crash" from the app menu to force an app crash.
 
-You can import this project into Android Studio if you so desire by selecting `Import Project (Eclipse ADT, Gradle, etc)` from the menu and then build as you would ordinarily. The token server setup is still important.
+[Read more](https://firebase.google.cn/docs/crashlytics/ndk-reports-new-sdk) about Android NDK crash reports.
```
Comment (berkus) [inline README.md]: Invalid url.

**Review by berkus — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (rusmonster) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (rusmonster) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (rusmonster) [reply README.md]: fixed

**Review by berkus — APPROVED** [review]

**Pull request merged**